src/Castle.Core/Castle.Core-VS2017.csproj

```diff
@@ -1,42 +1,49 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <Import Project="..\..\buildscripts\common.props"></Import>
```

Comment:
Tried loading editorconfig plugin but I think Resharper might be getting in the way. I have set my editor to use tabs.